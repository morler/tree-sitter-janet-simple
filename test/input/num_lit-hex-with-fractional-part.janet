0x09.1F