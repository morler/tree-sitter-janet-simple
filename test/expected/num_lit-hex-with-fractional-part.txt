(source [0, 0] - [0, 7]
  (num_lit [0, 0] - [0, 7]))
